src/inference/src/model_reader.cpp

```diff
@@ -111,6 +111,8 @@ std::shared_ptr<ov::Model> read_model(const std::string& modelPath,
                                       const std::string& binPath,
                                       const std::vector<ov::Extension::Ptr>& extensions,
                                       bool enable_mmap) {
+    OPENVINO_ASSERT(file_exists(modelPath.c_str()), "Model file does not exist at: ", modelPath);
```

Comment:
what if frontend is called directly? should we move it somewhere to base frontend class?

Reply:
Move check into frontends. As each have little different logic update base class not always works correctly.